samples/package.json

```diff
@@ -8,15 +8,28 @@
     "node": ">=8"
   },
   "repository": "googleapis/nodejs-asset",
-  "private": true,
   "nyc": {
     "exclude": [
       "**/*.test.js"
     ]
   },
-  "scripts": {},
+  "scripts": {
+    "test": "repo-tools test run --cmd ava -- -T 20s --verbose system-test/*.test.js"
+  },
   "dependencies": {
-    "@google-cloud/asset": "^0.1.0"
+    "@google-cloud/asset": "^0.1.0",
+    "@google-cloud/storage": "^2.3.0",
+    "express": "^4.16.4",
+    "uuid": "^3.3.2",
+    "yargs": "^12.0.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@google-cloud/nodejs-repo-tools": "^3.0.0",
+    "ava": "0.25.0",
```

Comment:
drop for mocha

Reply:
Dropping ava in both package.json results in below error when run command "npm run lint ./samples/system-test/*.js -- --fix":

~/github/nodejs-asset/samples/system-test/quickstart.test.js
  19:22  error  "ava" is extraneous  node/no-extraneous-require

✖ 1 problem (1 error, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/asset@0.1.0 lint: `eslint src/ samples/ system-test/ test/ "./samples/system-test/quickstart.test.js" "--fix"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/asset@0.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!    ~/.npm/_logs/2018-11-15T05_47_31_397Z-debug.log